### PR TITLE
Add Search option

### LINKED ISSUE #125: Search option
I believe we want to add a search option to posts.
my suggestion is to add a Search box to the navbar and search for a match in the post Location or the post description.
For now, the best thing I could think of is the use icontains search option with is equivalent to LIKE  operator in SQL.
@maayanhd @Shoval-Ovadia @Leeadar Any notes?

### PR DESCRIPTION
**Pull request by nadavspinco**
add search text filed and button to the navbar
add postListSearch to Post/views and to the URLs
change base.html colors style

closes #125

### CODE REVIEW

File: Post/views.py [hunk 1]
```diff
@@ -7,6 +7,15 @@
 from django.contrib import messages
 
 
+def postListSearch(request):
```
Comment (Yarboa) [inline Post/views.py:10]: Why is it Global function?
Are you going to add tests

Comment (nadavspinco) [reply Post/views.py:10]: @Yarboa 
Yes, I will add tests, I just wanted a mid-review about the search.
I forgot to add it under the class(basically there is no need, but I will for the order)

File: Post/tests/test_url_posts.py [hunk 2]
```diff
@@ -14,3 +14,7 @@ def test_post_detail_url_is_resolver(self):
     def test_createPost_url_is_resolved(self):
         url = reverse('createPost')
         assert resolve(url).func == CreateNewPost
+
+    def test_searchPost_url_is_resovled(self):
+        url = reverse('post_list_Search')
+        assert resolve(url).func == postListSearch
```
Comment (guy9050) [inline Post/tests/test_url_posts.py:20]: can you please explain this test?

File: pickATrip_django_apps/templates/pickATrip_django_apps/base.html [hunk 3]
```diff
@@ -71,7 +79,7 @@
 		{% endfor %}
 	{% endif %}
 
-	<div id ="content" class = "container" style="min-height: 83%">
+	<div id ="content" class = "container" style="min-height: 83%;>
```
Comment (guy9050) [inline pickATrip_django_apps/templates/pickATrip_django_apps/base.html:82]: unintended change?

File: Post/tests/test_url_posts.py (hunk 2 above)
Comment (nadavspinco) [reply Post/tests/test_url_posts.py:20]: @guy9050 yes.
This test checks that every URL is mapping to the view function we wanted to.

File: pickATrip_django_apps/templates/pickATrip_django_apps/base.html (hunk 3 above)
Comment (nadavspinco) [reply pickATrip_django_apps/templates/pickATrip_django_apps/base.html:82]: @guy9050 yes.
I will fix this.

Comment (nadavspinco) [reply pickATrip_django_apps/templates/pickATrip_django_apps/base.html]: Done.

**Comment by Leeadar** [comment]
image

I there a way that out names will stay at the bottom of page?

**Comment by nadavspinco** [comment]
> image
> 
> I there a way that out names will stay at the bottom of page?

Done.

**Review by maayanhd — CHANGES_REQUESTED** [review]
All for all it's very nice, I see it's working based on locations and descriptions, I don't think we need to change the feature, but maybe add something indicative like a grey default title like "type to search by location/description" style, that would be a good compromise I think .
Also maybe add a no posts found message for searches that fail.

**Review by Leeadar — APPROVED** [review]
LGTM

File: picATrip/settings.py [hunk 4]
```diff
@@ -135,3 +137,7 @@
 LOGIN_URL = 'login'
 
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
+
+MESSAGE_TAGS = {
+    messages.ERROR: 'danger',
```
Comment (nadavspinco) [inline picATrip/settings.py:142]: https://stackoverflow.com/questions/55202684/does-bootstrap-django-error-message-has-no-red-color

**Comment by nadavspinco** [comment]
I updated the PR and removed the new style. We will decide as a group on the style and make a new pr later for that.

**Review by maayanhd — APPROVED** [review]
LGTM

**Review by Shoval-Ovadia — APPROVED** [review]
LGTM

**Review by Leeadar — APPROVED** [review]
Nice!

**Review by Yarboa — APPROVED** [review]
LGTMT

**Pull request merged**